src/paste/deploy/config.py

```diff
@@ -84,8 +84,8 @@ def _pop_from(self, lst, conf):
         if conf is not None and popped is not conf:
             raise AssertionError(
                 "The config popped (%s) is not the same as the config "
-                "expected (%s)"
-                % (popped, conf))
+                "expected (%s)" % (popped, conf)
```

Comment:
No f string?

Reply:
I can manually switch it but pyupgrade didn't feel like doing it.